portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ABTestDXPSide.testcase

```diff
@@ -0,0 +1,525 @@
+			ACTests.terminateABTest(testName = ${nameABTest});
+		}
+
+		task ("Go to Tests") {
+			ACNavigation.goToTests();
+		}
+
+		task ("Assert AB test Terminated") {
+			ACUtils.viewGenericText(textValueList = ${nameABTest});
+
+			ACTests.statusABTest(status = "TERMINATED");
+		}
+
+		task ("Delete AB Test") {
+			ACTests.deleteABTest(testName = ${nameABTest});
+		}
+
+		task ("Go to Tests") {
+			ACNavigation.goToTests();
+		}
+
+		task ("Assert AB test Deleted") {
+			ACUtils.viewGenericText(textValueList = "There are no tests found.");
+		}
+	}
+
+	@description = "Story: LRAC-14101 | Automation ID: LRAC-14544 | Test Summary: Delete a terminated test in DXP"
+	@priority = 4
+	test CanDeleteTerminatedTestOnDXP {
+		task ("Create AB test") {
+			ABTest.createABTest(
+				goal = "Click",
+				testTitle = ${nameABTest});
+		}
+
+		task ("Select the button element added to the page for AB test") {
+			ABTest.selectElementID();
+		}
+
+		task ("Create variable") {
+			ABTest.createVariant(variantName = "Variant name");
+		}
+
+		task ("Click on Review and Run test") {
+			ABTest.runTest();
+		}
+
+		task ("Go to AC > Tests") {
+			ACUtils.launchAC();
+
+			ACProperties.switchProperty(propertyName = ${assignedPropertyName});
+
+			ACNavigation.goToTests();
+		}
+
+		task ("Check that the created AB test appears in the list") {
+			ACUtils.viewNameList(nameList = ${nameABTest});
+		}
+
+		task ("Go to DXP > Go to AB Test Side Panel") {
+			ACUtils.navigateToSitePage(
+				pageName = "AC Page",
+				siteName = "Site Name");
+
+			ABTest.openABTestPanel();
+		}
+
+		task ("Terminate the test") {
+			ABTest.terminateTest();
+		}
+
+		task ("Delete AB Test") {
+			ABTest.deleteABTest();
+		}
+
+		task ("Assert the empty state after the test is deleted") {
+			ABTest.assertDXPABTestEmptyState();
+		}
+
+		task ("Go to AC > Tests") {
+			ACUtils.launchAC();
+
+			ACProperties.switchProperty(propertyName = ${assignedPropertyName});
+
+			ACNavigation.goToTests();
+		}
+
+		task ("Check that the empty state message appears in the list") {
+			ACUtils.viewNoResultsMessage(noResultMessage = "There are no tests found.");
+		}
+	}
+
+	@description = "Story: LRAC-14070 | Automation ID: LRAC-14574 | Test Summary: Can publish control from a terminated test"
+	@priority = 5
+	test CanPublishControlFromTerminatedTest {
+		task ("Edit the variant created") {
+			ABTest.editVariant(variantName = "Variant name");
+		}
+
+		task ("Add a heading to the page to diferentiate from the control") {
+			PageEditor.addFragment(
+				collectionName = "Basic Components",
+				fragmentName = "Heading");
+		}
+
+		task ("Click to save the variant") {
+			ABTest.saveVariant();
+		}
+
+		task ("Click to open the AB test panel") {
+			ABTest.openABTestPanel();
+
+			task ("Click to review and run the test") {
+				ABTest.runTest();
+			}
+		}
+
+		task ("Click to terminate the test") {
+			Button.click(button = "Terminate Test");
+
+			ABTest.assertTerminateTest();
+		}
```

Comment:
Hey @manuferreira, I don't think it's necessary to check the modal text in these tests since it's not the purpose of the test. We have this case that already check this modal:

- CheckTerminateButtonRedirectingToDXP

Putting all the tests to make this view may not be very good, since if we have a change in the text, then all the tests will fail. As we have this test for the purpose of checking the redirect, I think it is the best case to validate this message whenever it is executed. What do you think?

I think you could use:

```
task ("Terminate AB Test") {
	ACTests.terminateABTest(testName = ${nameABTest});
}
```

Reply:
Agree and since this is being pointed out, in some moment we will have to review a few of AC tests, since some of them are doing what they are not supposed to do (doing assertions and stuff when it's not the aim of the test).

Reply:
I don't think I can use this macro when I'm terminating the test starting on DXP side. I believe this macro can be only used to terminate the test starting on AC -> then going to DXP. :thinking: 
I think I should use this one:
`ABTest.terminateTest();`
Also, I noticed some other inconsistencies in other tests, so I'm also correcting them.

Reply:
Oh yeah, you're right. I copied the wrong file name when I wrote it here

Reply:
https://github.com/liferay-ac/liferay-portal/pull/729/commits/33d55e7aee456c1ce89d472e89a8030e7bb36e0c